frontend/components.js

```diff
@@ -275,7 +275,7 @@ class UnlockClass extends Component {
             class: 'styled-input-nodiv styled-unlock-input',
             id: 'mnemonic-submitted',
             name: 'mnemonic-submitted',
-            placeholder: 'Enter twelve-word mnemonic',
+            placeholder: 'Enter 12-word mnemonic or paper wallet mnemonic',
```

Comment:
Just wanted to point out that for example is using 12, 18 or 24 words mnemonic.
Also the paper wallet addition makes the placeholder too long and on iphone5 will not be visible and since it is a placeholder you can not even scroll it.

I suggest to put short paragraph above the input with short note same as in the trezor tab.